### ISSUE TITLE
Files specified in dependencies get copied twice

### ISSUE THREAD

**Issue by saviomuc** [trulia/hologram#197]
For instance when I create  a dependency

destination:
./build
dependencies:
  - ./css

the expected output should be 
/build/css/

This works fine for the first time. However if you run 'hologram' a second time it will copy the dependency files to 
/build/css/css

I am using hologram 1.3, ruby 2.1.2p95 on Mac Os X 10.9.2

**Comment by BenSpace48** [comment]
I'm getting the same issue, when I first generate the files it is fine. The files are generated to docs/css, but the second time I generate they go to docs/css/css and I've no idea why. It is getting irritating having to delete the directory every time as I can't use grunt clean on this project. 

#### Relevent config

```
# The directory containing the source files to parse recursively
source: ../scss

# The directory that hologram will build to
destination: ./docs

dependencies:
  - ../css
  - ../scss
  - ../fonts
  - ../img
  - ../js
```

**Comment by ventrebleu** [comment]
Same here.

**Comment by jon64digital** [comment]
Yep, exactly the same issue, using gulp-hologram

**Comment by jdcantrell** [comment]
This should be fixed in 1.3.1 which is now available.
